### PR TITLE
Consolidate option names

### PR DESCRIPTION
**Pull request by cortesi**
- Onboarding app options to onboarding*
- Console-specific options to console_*
- Web-specific options to web_*

### CODE REVIEW

**Comment by Kriechi** [comment]
Prefixing looks kinda weird - any thoughts on using a proper class / wrapper object?

**Comment by mhils** [comment]
The bigger question is how we integrate external addons. In any case, this is a great intermediary step to actually prefix everything with what it belongs to.

File: mitmproxy/tools/cmdline.py [hunk 1]
```diff
@@ -568,23 +568,23 @@ def proxy_ssl_options(parser):
 def onboarding_app(parser):
     group = parser.add_argument_group("Onboarding App")
     group.add_argument(
-        "--noapp",
-        action="store_false", dest="app",
+        "--noonboarding",
```
Comment (Kriechi) [inline mitmproxy/tools/cmdline.py]: I'd prefer `--no-onboarding` - IMHO this is a more common way to negate options.

**Comment by cortesi** [comment]
We toyed with separate classes for this before, and it introduces a lot of complexity. I think prefixing is fine for now. We could add some sugar to the config file if we want so that a ```console:``` section automatically translates into a ```console_``` prefix to make user configuration a bit nicer.

File: mitmproxy/tools/cmdline.py (hunk 1 above)
Comment (mhils) [inline mitmproxy/tools/cmdline.py]: 🕛🏄 it is!

Comment (cortesi) [inline mitmproxy/tools/cmdline.py]: Done.

**Comment by Kriechi** [comment]
Ok - sounds good to me!

File: mitmproxy/tools/cmdline.py [hunk 2]
```diff
@@ -568,23 +568,23 @@ def proxy_ssl_options(parser):
 def onboarding_app(parser):
     group = parser.add_argument_group("Onboarding App")
     group.add_argument(
-        "--noapp",
-        action="store_false", dest="app",
+        "--no-onboarding",
+        action="store_false", dest="onboarding",
         help="Disable the mitmproxy onboarding app."
```
Comment (Kriechi) [inline mitmproxy/tools/cmdline.py]: Is everybody familiar with the onboarding app in general?
Maybe We should add a one sentence description of what the purpose is?

**Comment by Kriechi** [comment]
@cortesi please add more tests to increase coverage of optmanager back to 100%!

**Pull request merged**